utils.js

```diff
@@ -0,0 +1,10 @@
+import { t } from 'testcafe';
+
+export default class NamePriceCheck{
+    
+    async nameAndPriceCheck(name1,name2,price1,price2){
+        await t.expect(name1).eql(name2, 'The name of the game is not the same');
```

Comment:
instead of checking field by field you can compare two game objects (which will have name and price fields) using t.expect.eql
and do the comparation in the test itself so that it be more transparent what exactly is checked and when